### ISSUE TITLE
duplicate after cursor on edges

### ISSUE THREAD

**Issue by darwinhace** [blb-ventures/strawberry-django-plus#4]
"after cursor" is still included in the results of edges, as far as I know "after cursor" is no longer included in the results of edges.

![2022-02-07_17-13](https://user-images.githubusercontent.com/76155308/152768445-f98d3fb9-f128-454e-aafa-f4ab6f70400b.png)

**Comment by bellini666** [comment]
Hey @darwinhace , thanks for reporting this. Just fixed the issue and it it being released as I write this to you on version 0.8.

**Comment by darwinhace** [comment]
hey @bellini666, thx for the response, after is working now, but now when using arguments {last: 2, before: "..."}, one node missing from edges:

before using arguments:
![image](https://user-images.githubusercontent.com/76155308/152907069-9db9df41-3baf-4bbe-8c7e-52145c2585a0.png)

after using arguments:
![image](https://user-images.githubusercontent.com/76155308/152907165-0c5c2d7e-685e-4ff0-9402-1a5e703ab96f.png)

cursor: "YXJyYXljb25uZWN0aW9uOjE=" missing from edges,

when I remove "-1" from this code it is work as expected

```
  if before:
            before_type, before_parsed = from_base64(before)
            assert before_type == _connection_typename
            end = min(end, int(before_parsed) - 1)
```
